packaging/packaging

```diff
@@ -0,0 +1 @@
+.
```

Comment:
Shouldn't this file be removed?  Not sure why packaging/install includes this file.

Reply:
Yes.  Removed.  And daos-stack/packaging#32.